chart/templates/otel-auto-instrumentation.yaml

```diff
@@ -2,7 +2,10 @@
 apiVersion: opentelemetry.io/v1alpha1
 kind: Instrumentation
 metadata:
-  name: tobs-auto-instrumentation
+  name: {{ .Release.Name }}-auto-instrumentation
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade,pre-delete
```

Comment:
Those are needed to allow installing resource after pods are ready (using `--wait` flag from helm).